lib/r10k/module_loader/puppetfile.rb

```diff
@@ -41,11 +45,14 @@ def initialize(basedir:,
         @managed_directories = []
         @desired_contents = []
         @purge_exclusions = []
+
+        logger.info _("Using Puppetfile '%{puppetfile}'") % {puppetfile: @puppetfile_path}
+        logger.info _("Using moduledir '%{moduledir}'") % {moduledir: @moduledir}
```

Comment:
I think this should be at debug.

Reply:
I was thinking that too.

Reply:
Both or just the `moduledir` one? Either way is fine with me, just that the `puppetfile` one was at `info` before, which is why I put them both at that level here, but I don't disagree that debug makes sense.

Reply:
I'd just do the moduledir one, I think they both feel like debug statements but since the puppetfile one was info we should probably unfortunately keep it there.

Reply:
updated